test/Directory.Build.props

```diff
@@ -2,6 +2,8 @@
   <Import Project="..\Directory.Build.props" />
 
   <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <IsPackable>false</IsPackable>
```

Comment:
What does this do?

Reply:
It's in the default `xunit` template.  It tell `dotnet pack` to skip this project and don't produce a NuGet package.